package.json

```diff
@@ -60,7 +60,9 @@
 		"chat"
 	],
 	"scripts": {
-		"start": "meteor npm i && meteor",
+		"start": "meteor npm i && meteor run",
+		"debug": "meteor run --inspect",
```

Comment:
This was actually independent of this PR, but somehow made its way into it. It does no harm. Please remove if not favored.

Reply:
removed